homeassistant/components/enocean/services.py

```diff
@@ -0,0 +1,287 @@
+"""EnOcean services."""
+from __future__ import annotations
+
+import logging
+import queue
+import time
+
+from enocean import utils
+from enocean.communicators import Communicator
+from enocean.protocol.constants import PACKET, RORG
+from enocean.protocol.packet import Packet, UTETeachInPacket
+import voluptuous as vol
+
+from homeassistant.core import HomeAssistant, ServiceCall
+
+from .const import DOMAIN
+from .teachin import FourBsTeachInHandler, TeachInHandler, UteTeachInHandler
+from .utils import get_communicator_reference, hex_to_list
+
+TEACH_IN_DEVICE = "teach_in_device"  # service name
+SERVICE_CALL_ATTR_TEACH_IN_SECONDS = "teach_in_time"
+SERVICE_CALL_ATTR_TEACH_IN_SECONDS_DEFAULT_VALUE_STR = "60"
+SERVICE_CALL_ATTR_TEACH_IN_SECONDS_DEFAULT_VALUE = 60
+SERVICE_CALL_ATTR_TEACH_IN_BASE_ID_TO_USE = "teach_in_base_id"
```

Comment:
Shouldn't this match services.yaml ?
```suggestion
SERVICE_CALL_ATTR_TEACH_IN_BASE_ID_TO_USE = "base_id"
```